src/Version.h

```diff
@@ -14,9 +14,9 @@
 #ifndef SNOWCRASH_VERSION_H
 #define SNOWCRASH_VERSION_H
 
-#define SNOWCRASH_MAJOR_VERSION 0
-#define SNOWCRASH_MINOR_VERSION 16
-#define SNOWCRASH_PATCH_VERSION 1
+#define SNOWCRASH_MAJOR_VERSION 1
```

Comment:
Main question is whether we need this versioning here – as it was used mainly for the CLI tool